frequency-plans.yml

```diff
@@ -186,7 +369,7 @@
 
 - id: AS_923
   band-id: AS_923
-  name: Asia 915–928 MHz (AS923 Group 1) with only default channels
+  name: Asia 915-928 MHz (AS923 Group 1) with only default channels
```

Comment:
Different unicode character?

Reply:
Indeed, the old one was not the ASCII character.